app/models/Errors.scala

```diff
@@ -0,0 +1,7 @@
+package models
+
+/**
+  * Error when uploading CSV
+  * @param msg the error message
+  */
+case class ErrorUploadCsv(msg: String)
```

Comment:
You could also make it be a "value class" which will make it a `String` at runtime but a `ErrorUploadCsv` at compile time

```scala
final case class ErrorUploadCsv(msg: String) extends AnyVal
```